library/Zend/Console/Prompt/Checkbox.php

```diff
@@ -0,0 +1,195 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+namespace Zend\Console\Prompt;
+
+use Zend\Console\Exception;
+use Zend\Stdlib\ArrayUtils;
+
+final class Checkbox extends AbstractPrompt
+{
+    /**
+     * @var string
+     */
+    private $promptText = 'Please select an option (Enter to finish) ';
```

Comment:
Default values are also not needed anymore if `__construct` deals with all of them ;-)